### DISCUSSION TITLE
Removing Theme Menu Actions

### DISCUSSION THREAD

**Discussion by maheshrajrp** [eclipse-theia/theia#9897] [Q&A] [answered]
Hi, 
I've used unregisterMenuAction to remove the ability to change the theme. However, the same pops up while using the Quick Command Functionality and so I tried
 `registry.unregisterCommand(CommonCommands.SELECT_ICON_THEME);`

However, this doesn't allow the application to start. It would be helpful if you could assist me in understanding, what went wrong and how to fix.

Thanks

**Comment by msujew** [comment, marked as answer]
@maheshrajrp This is a known issue and will be resolved by #9886. I don't think there is a workaround for this currently.

**Reply by maheshrajrp** [reply]
Thanks for the prompt reply, @msujew . I'll follow the issue.
